### ISSUE TITLE
Switch to use equal area projection with geoblaze

### ISSUE THREAD

**Issue by twelch** [seasketch/geoprocessing#162]
Currently only EPSG 4326 is used. The problem with 4326 spherical lat/lon is the raster cell area varies. It's not easy to take a count of the number of raster cells overlapping with a sketch for example and multiply by cell resolution^2 to get area.

GP tasks:
- [x] Discuss equal area projection to be used, with consistent cell area.  This could be UTM, with varying zone depending on the project, or a global equal area projection.  Decision to **use EPSG-6933 (NSIDC EASE-Grid 2.0 Global)**
- [x] write unit test in gp library that uses the alternate projection and verifies proper function
- [ ] Add area to toolbox function as metric output
- [x] Migrate a project to use it

**Comment by DanielJDufour** [comment]
I'm wondering if it might be useful to explore adopting [geowarp](https://github.com/danieljdufour/geowarp) which is roughly like a gdalwarp/gdaltranslate in pure JavaScript.  Are you thinking of doing reprojection outside of geoblaze or inside?

In theory, it would be possible to add an equal_area flag to geoblaze, which means geoblaze would automatically reproject/resample values internally before calculating statistics.  However, this would add some more complexity to geoblaze, so it would have to be done carefully.  It would be a fun challenge though and I definitely see the value in doing so.

tl;dr: if you are leaning towards reprojecting as a preprocessing step and passing the reprojected georaster to geoblaze, I would definitely be okay with that.

**Comment by twelch** [comment]
There are gdal data prep scripts that take on reprojecting already.

**Comment by DanielJDufour** [comment]
Awesome!

**Comment by twelch** [comment]
if polygon crossing antimeridian, it reprojects to SRS of raster, may be an issue here to look into.

**Comment by twelch** [comment]
Can assume geoprocessing framework will normalize on a single projection.

**Comment by twelch** [comment]
reprojection library used by geoblaze - https://github.com/danieljdufour/reproject-geojson

**Comment by twelch** [comment]
The one equal area projection to use could potentially be  World Eckert IV - ESRI:54012

**Comment by twelch** [comment]
Decision is (still) to use EPSG-6933 (NSIDC EASE-Grid 2.0 Global

**Comment by twelch** [comment]
Resolved via #259
